README.md

```diff
@@ -33,7 +33,8 @@ This section describes the nested topology design implemented by this solution.
 ![alt text](architecture/nested-topology-hld.png "Nested Toplogy")
 
 At the core of the nested topology design, we have proxies (currently Squid) which broker the connections between each hypothetical ISA-95 level (Level 2,3,4 in this instance).
-These proxies prevent workloads and Arc agents running at lower levels from connecting to the outside world directly, allowing the traffic to be managed/controlled via proxy configuration at each level. We are currently enhancing this functionality by enforcing data plane to traverse via proxy as well, as we only have Arc configured for it for now.
+These proxies prevent workloads and Arc agents running at lower levels from connecting to the outside world directly, allowing the traffic to be managed/controlled via proxy configuration at each level.
+We are currently implementing functionality that will force the data plane to traverse via the proxy as well, but only Arc is currently configured.
```

Comment:
@katriendg thinking if we are still unsure re use of proxy for data plane?

Reply:
Also reflecting on this paragraph, we could also mention we are working on ensuring Container Registry downloads will be proxied too. This is a task on our board to review usage of Connected Container Registry, today it's still going directly to the public GH endpoints (or ACR when deploying the developer version).

Reply:
```suggestion
```